ccmlib/node.py

```diff
@@ -1404,12 +1404,13 @@ def import_config_files(self):
     def import_dse_config_files(self):
         raise common.ArgumentError('Cannot import DSE configuration files on a Cassandra node')
 
-    def copy_config_files(self):
+    def copy_config_files(self, clobber=False):
```

Comment:
if there any code that is going to call this with `clobber=True` ?

Reply:
I don't think so, but I wanted to easily allow it in case it's needed.
That said, it can always be enabled when needed

Reply:
weird that github actions didn't triggered